code/modules/research/techweb/techweb_types.dm

```diff
@@ -0,0 +1,95 @@
+/**
+ * Global Science techweb for RND consoles
+ */
+/datum/techweb/science
+	id = "SCIENCE"
+	organization = "Nanotrasen"
+
+//When something is researched, triggers the proc for this techweb only
+/datum/techweb/science/research_node(datum/techweb_node/node, force = FALSE, auto_adjust_cost = TRUE, get_that_dosh = TRUE)
+	. = ..()
+	if(.)
+		node.on_research()
+
+/**
+ * Admin techweb that has everything unlocked by default
+ */
+/datum/techweb/admin
+	id = "ADMIN"
+	organization = "CentCom"
+
+/datum/techweb/admin/New()
+	. = ..()
+	for(var/i in SSresearch.techweb_nodes)
+		var/datum/techweb_node/TN = SSresearch.techweb_nodes[i]
```

Comment:
can you use `as anything` here

Reply:
We don't need ``as anything`` because we're not casting any type as ``i``

Reply:
sry i mean a for .. in loop rather, `for(var/datum/techweb_node/node in SSresearch.techweb_nodes)`

Reply:
No we can't, because it's an assosciate list